x/fungible/keeper/zrc20_lock_token.go

```diff
@@ -0,0 +1,82 @@
+package keeper
```

Comment:
On the organization side, I think it would be simpler to have a single file in `fungible` for all ZRC20 mapping logic.

Can be called `zrc20_cosmos_coin_mapping.go`

Reply:
Modified here: 445b5c3287eeec113488bbfec6f8bcd26ee83413